### DISCUSSION TITLE
Dynamic row style class

### DISCUSSION THREAD

**Discussion by david-zed** [primefaces/community#2114] [PrimeFaces] [answered]
Hi, I am new to this so please be patient. I have a p:dataTable where the selected row is notified to the server using p:ajax call but what I would like to do is to set the background colour of the selected row so the user can see straight away that it is selected. At the moment the row simply does not change although the server is notified of the selection

**Comment by melloware** [comment, marked as answer]
Example of what you want to do: https://www.primefaces.org/showcase/ui/data/datatable/rowColor.xhtml?jfwid=50d9e

**Comment by david-zed** [comment]
Hi thanks for the update, much appreciated
